aip/general/0234.md

```diff
@@ -42,6 +42,8 @@ rpc BatchUpdateBooks(BatchUpdateBooksRequest) returns (BatchUpdateBooksResponse)
   succeed for all resources (no partial success).
   - If the operation covers multiple locations and at least one location is
     down, the operation **must** fail.
+- The operation **must** execute independent updates based on a point-in-time
+  snapshot. It is thus independent of the order of requests.
```

Comment:
Could we just say,

> The operation **must** execute independent updates independent of the order of requests.

?

Reply:
The point about the snapshot was to clarify the behavior with some other angle.  I'm seeing the brevity of this framing, but would also like to know the opinions of others before changing.